<!doctype html>
<html class="no-js" lang="en" data-content_root="./">
  <head><meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width,initial-scale=1"/>
    <meta name="color-scheme" content="light dark"><meta name="viewport" content="width=device-width, initial-scale=1" />
<link rel="index" title="Index" href="genindex.html" /><link rel="search" title="Search" href="search.html" /><link rel="next" title="Benchmark data sets" href="Datasets_Benchmark.html" /><link rel="prev" title="Unconventional methods" href="Acquisition_Unconventional.html" />

    <!-- Generated with Sphinx 7.2.6 and Furo 2023.09.10 -->
        <title>DIA data analysis - Appendix for a DIA review</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=a746c00c" />
    <link rel="stylesheet" type="text/css" href="_static/styles/furo.css?v=135e06be" />
    <link rel="stylesheet" type="text/css" href="_static/tabs.css?v=4c969af8" />
    <link rel="stylesheet" type="text/css" href="_static/styles/furo-extensions.css?v=36a5483c" />
    
    


<style>
  body {
    --color-code-background: #f8f8f8;
  --color-code-foreground: black;
  
  }
  @media not print {
    body[data-theme="dark"] {
      --color-code-background: #202020;
  --color-code-foreground: #d0d0d0;
  
    }
    @media (prefers-color-scheme: dark) {
      body:not([data-theme="light"]) {
        --color-code-background: #202020;
  --color-code-foreground: #d0d0d0;
  
      }
    }
  }
</style></head>
  <body>
    
    <script>
      document.body.dataset.theme = localStorage.getItem("theme") || "auto";
    </script>
    

<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
  <symbol id="svg-toc" viewBox="0 0 24 24">
    <title>Contents</title>
    <svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 1024 1024">
      <path d="M408 442h480c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8H408c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8zm-8 204c0 4.4 3.6 8 8 8h480c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8H408c-4.4 0-8 3.6-8 8v56zm504-486H120c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm0 632H120c-4.4 0-8 3.6-8 8v56c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zM115.4 518.9L271.7 642c5.8 4.6 14.4.5 14.4-6.9V388.9c0-7.4-8.5-11.5-14.4-6.9L115.4 505.1a8.74 8.74 0 0 0 0 13.8z"/>
    </svg>
  </symbol>
  <symbol id="svg-menu" viewBox="0 0 24 24">
    <title>Menu</title>
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor"
      stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather-menu">
      <line x1="3" y1="12" x2="21" y2="12"></line>
      <line x1="3" y1="6" x2="21" y2="6"></line>
      <line x1="3" y1="18" x2="21" y2="18"></line>
    </svg>
  </symbol>
  <symbol id="svg-arrow-right" viewBox="0 0 24 24">
    <title>Expand</title>
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor"
      stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather-chevron-right">
      <polyline points="9 18 15 12 9 6"></polyline>
    </svg>
  </symbol>
  <symbol id="svg-sun" viewBox="0 0 24 24">
    <title>Light mode</title>
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor"
      stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" class="feather-sun">
      <circle cx="12" cy="12" r="5"></circle>
      <line x1="12" y1="1" x2="12" y2="3"></line>
      <line x1="12" y1="21" x2="12" y2="23"></line>
      <line x1="4.22" y1="4.22" x2="5.64" y2="5.64"></line>
      <line x1="18.36" y1="18.36" x2="19.78" y2="19.78"></line>
      <line x1="1" y1="12" x2="3" y2="12"></line>
      <line x1="21" y1="12" x2="23" y2="12"></line>
      <line x1="4.22" y1="19.78" x2="5.64" y2="18.36"></line>
      <line x1="18.36" y1="5.64" x2="19.78" y2="4.22"></line>
    </svg>
  </symbol>
  <symbol id="svg-moon" viewBox="0 0 24 24">
    <title>Dark mode</title>
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor"
      stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" class="icon-tabler-moon">
      <path stroke="none" d="M0 0h24v24H0z" fill="none" />
      <path d="M12 3c.132 0 .263 0 .393 0a7.5 7.5 0 0 0 7.92 12.446a9 9 0 1 1 -8.313 -12.454z" />
    </svg>
  </symbol>
  <symbol id="svg-sun-half" viewBox="0 0 24 24">
    <title>Auto light/dark mode</title>
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="none" stroke="currentColor"
      stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round" class="icon-tabler-shadow">
      <path stroke="none" d="M0 0h24v24H0z" fill="none"/>
      <circle cx="12" cy="12" r="9" />
      <path d="M13 12h5" />
      <path d="M13 15h4" />
      <path d="M13 18h1" />
      <path d="M13 9h4" />
      <path d="M13 6h1" />
    </svg>
  </symbol>
</svg>

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation">
<input type="checkbox" class="sidebar-toggle" name="__toc" id="__toc">
<label class="overlay sidebar-overlay" for="__navigation">
  <div class="visually-hidden">Hide navigation sidebar</div>
</label>
<label class="overlay toc-overlay" for="__toc">
  <div class="visually-hidden">Hide table of contents sidebar</div>
</label>



<div class="page">
  <header class="mobile-header">
    <div class="header-left">
      <label class="nav-overlay-icon" for="__navigation">
        <div class="visually-hidden">Toggle site navigation sidebar</div>
        <i class="icon"><svg><use href="#svg-menu"></use></svg></i>
      </label>
    </div>
    <div class="header-center">
      <a href="index.html"><div class="brand">Appendix for a DIA review</div></a>
    </div>
    <div class="header-right">
      <div class="theme-toggle-container theme-toggle-header">
        <button class="theme-toggle">
          <div class="visually-hidden">Toggle Light / Dark / Auto color theme</div>
          <svg class="theme-icon-when-auto"><use href="#svg-sun-half"></use></svg>
          <svg class="theme-icon-when-dark"><use href="#svg-moon"></use></svg>
          <svg class="theme-icon-when-light"><use href="#svg-sun"></use></svg>
        </button>
      </div>
      <label class="toc-overlay-icon toc-header-icon" for="__toc">
        <div class="visually-hidden">Toggle table of contents sidebar</div>
        <i class="icon"><svg><use href="#svg-toc"></use></svg></i>
      </label>
    </div>
  </header>
  <aside class="sidebar-drawer">
    <div class="sidebar-container">
      
      <div class="sidebar-sticky"><a class="sidebar-brand" href="index.html">
  
  
  <span class="sidebar-brand-text">Appendix for a DIA review</span>
  
</a><form class="sidebar-search-container" method="get" action="search.html" role="search">
  <input class="sidebar-search" placeholder="Search" name="q" aria-label="Search">
  <input type="hidden" name="check_keywords" value="yes">
  <input type="hidden" name="area" value="default">
</form>
<div id="searchbox"></div><div class="sidebar-scroll"><div class="sidebar-tree">
  <p class="caption" role="heading"><span class="caption-text">Acquisition:</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Acquisition_Full_Scan_DIA.html">Full scan DIA</a></li>
<li class="toctree-l1"><a class="reference internal" href="Acquisition_Windowed_DIA.html">Windowed DIA</a></li>
<li class="toctree-l1"><a class="reference internal" href="Acquisition_PASEF_DIA.html">PASEF enhanced DIA</a></li>
<li class="toctree-l1"><a class="reference internal" href="Acquisition_Unconventional.html">Unconventional methods</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Software:</span></p>
<ul class="current">
<li class="toctree-l1 current current-page"><a class="current reference internal" href="#">DIA data analysis</a></li>
</ul>
<p class="caption" role="heading"><span class="caption-text">Datasets:</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="Datasets_Benchmark.html">Benchmark data sets</a></li>
</ul>

</div>
</div>

      </div>
      
    </div>
  </aside>
  <div class="main">
    <div class="content">
      <div class="article-container">
        <a href="#" class="back-to-top muted-link">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
            <path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12z"></path>
          </svg>
          <span>Back to top</span>
        </a>
        <div class="content-icon-container">
          
<div class="theme-toggle-container theme-toggle-content">
            <button class="theme-toggle">
              <div class="visually-hidden">Toggle Light / Dark / Auto color theme</div>
              <svg class="theme-icon-when-auto"><use href="#svg-sun-half"></use></svg>
              <svg class="theme-icon-when-dark"><use href="#svg-moon"></use></svg>
              <svg class="theme-icon-when-light"><use href="#svg-sun"></use></svg>
            </button>
          </div>
          <label class="toc-overlay-icon toc-content-icon" for="__toc">
            <div class="visually-hidden">Toggle table of contents sidebar</div>
            <i class="icon"><svg><use href="#svg-toc"></use></svg></i>
          </label>
        </div>
        <article role="main">
          <section id="dia-data-analysis">
<h1>DIA data analysis<a class="headerlink" href="#dia-data-analysis" title="Link to this heading">#</a></h1>
<section id="overall-summary">
<h2>Overall summary<a class="headerlink" href="#overall-summary" title="Link to this heading">#</a></h2>
<div class="table-wrapper docutils container" id="table">
<table class="docutils align-default" id="table">
<thead>
<tr class="row-odd"><th class="head"><p>Name</p></th>
<th class="head"><p>Pub date</p></th>
<th class="head"><p>Dev stat</p></th>
<th class="head"><p>Stra.</p></th>
<th class="head"><p>x-first</p></th>
<th class="head"><p>Interface</p></th>
<th class="head"><p>Acce.</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Ion Accounting <a class="footnote-reference brackets" href="#ionaccounting" id="id1" role="doc-noteref"><span class="fn-bracket">[</span>1<span class="fn-bracket">]</span></a></p></td>
<td><p>2009 Mar.</p></td>
<td><p>/</p></td>
<td><p>R</p></td>
<td><p>/</p></td>
<td><p>/</p></td>
<td><p>C</p></td>
</tr>
<tr class="row-odd"><td><p>ETISEQ <a class="footnote-reference brackets" href="#id27" id="id2" role="doc-noteref"><span class="fn-bracket">[</span>2<span class="fn-bracket">]</span></a></p></td>
<td><p>2009 Aug.</p></td>
<td><p>Not</p></td>
<td><p>R</p></td>
<td><p>/</p></td>
<td><p>C</p></td>
<td><p>F</p></td>
</tr>
<tr class="row-even"><td><p>DeMux <a class="footnote-reference brackets" href="#id28" id="id3" role="doc-noteref"><span class="fn-bracket">[</span>3<span class="fn-bracket">]</span></a></p></td>
<td><p>2009 Dec.</p></td>
<td><p>/</p></td>
<td><p>R</p></td>
<td><p>/</p></td>
<td><p>/</p></td>
<td><p>R</p></td>
</tr>
<tr class="row-odd"><td><p>Skyline <a class="footnote-reference brackets" href="#id29" id="id4" role="doc-noteref"><span class="fn-bracket">[</span>4<span class="fn-bracket">]</span></a></p></td>
<td><p>2010 Apr.</p></td>
<td><p>Active</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C &amp; G</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>FT-ARM <a class="footnote-reference brackets" href="#ftarm" id="id5" role="doc-noteref"><span class="fn-bracket">[</span>9<span class="fn-bracket">]</span></a></p></td>
<td><p>2012 Jan.</p></td>
<td><p>Not</p></td>
<td><p>S/L</p></td>
<td><p>S</p></td>
<td><p>C &amp; G</p></td>
<td><p>F</p></td>
</tr>
<tr class="row-odd"><td><p>Spectronaut <a class="footnote-reference brackets" href="#id30" id="id6" role="doc-noteref"><span class="fn-bracket">[</span>10<span class="fn-bracket">]</span></a></p></td>
<td><p>2013</p></td>
<td><p>Active</p></td>
<td><p>R,L</p></td>
<td><p>C</p></td>
<td><p>C &amp; G</p></td>
<td><p>C</p></td>
</tr>
<tr class="row-even"><td><p>OpenSWATH <a class="footnote-reference brackets" href="#id31" id="id7" role="doc-noteref"><span class="fn-bracket">[</span>12<span class="fn-bracket">]</span></a></p></td>
<td><p>2014 Mar.</p></td>
<td><p>Active</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C &amp; G</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>DIANA <a class="footnote-reference brackets" href="#id32" id="id8" role="doc-noteref"><span class="fn-bracket">[</span>13<span class="fn-bracket">]</span></a></p></td>
<td><p>2014 Oct.</p></td>
<td><p>Not</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>DIA-Umpire <a class="footnote-reference brackets" href="#diaumpire" id="id9" role="doc-noteref"><span class="fn-bracket">[</span>14<span class="fn-bracket">]</span></a></p></td>
<td><p>2015 Jan.</p></td>
<td><p>Active</p></td>
<td><p>R,L</p></td>
<td><p>/,C</p></td>
<td><p>C &amp; G</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>SWATHProphet <a class="footnote-reference brackets" href="#id33" id="id10" role="doc-noteref"><span class="fn-bracket">[</span>16<span class="fn-bracket">]</span></a></p></td>
<td><p>2015 Feb.</p></td>
<td><p>Not</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>Group-DIA  <a class="footnote-reference brackets" href="#groupdia" id="id11" role="doc-noteref"><span class="fn-bracket">[</span>17<span class="fn-bracket">]</span></a></p></td>
<td><p>2015 Oct.</p></td>
<td><p>Not</p></td>
<td><p>R,L</p></td>
<td><p>/,C</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>MSPLIT-DIA  <a class="footnote-reference brackets" href="#msplitdia" id="id12" role="doc-noteref"><span class="fn-bracket">[</span>18<span class="fn-bracket">]</span></a></p></td>
<td><p>2015 Nov.</p></td>
<td><p>Not</p></td>
<td><p>L</p></td>
<td><p>S</p></td>
<td><p>C</p></td>
<td><p>F</p></td>
</tr>
<tr class="row-even"><td><p>PECAN (Walnut) <a class="footnote-reference brackets" href="#pecan" id="id13" role="doc-noteref"><span class="fn-bracket">[</span>19<span class="fn-bracket">]</span></a></p></td>
<td><p>2017 Aug.</p></td>
<td><p>Active</p></td>
<td><p>S</p></td>
<td><p>S</p></td>
<td><p>C &amp; G</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>Specter <a class="footnote-reference brackets" href="#id34" id="id14" role="doc-noteref"><span class="fn-bracket">[</span>20<span class="fn-bracket">]</span></a></p></td>
<td><p>2018 Apr.</p></td>
<td><p>Not</p></td>
<td><p>L</p></td>
<td><p>S</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>EncyclopeDIA <a class="footnote-reference brackets" href="#id35" id="id15" role="doc-noteref"><span class="fn-bracket">[</span>21<span class="fn-bracket">]</span></a></p></td>
<td><p>2018 Dec.</p></td>
<td><p>Active</p></td>
<td><p>L</p></td>
<td><p>S</p></td>
<td><p>C &amp; G</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>DeepNovo-DIA <a class="footnote-reference brackets" href="#deepnovodia" id="id16" role="doc-noteref"><span class="fn-bracket">[</span>23<span class="fn-bracket">]</span></a></p></td>
<td><p>2018 Dec.</p></td>
<td><p>Not</p></td>
<td><p>D</p></td>
<td><p>C</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>PEAKS <a class="footnote-reference brackets" href="#id36" id="id17" role="doc-noteref"><span class="fn-bracket">[</span>24<span class="fn-bracket">]</span></a></p></td>
<td><p>2018</p></td>
<td><p>Active</p></td>
<td><p>S,L,D</p></td>
<td><p>N</p></td>
<td><p>C &amp; G</p></td>
<td><p>C</p></td>
</tr>
<tr class="row-odd"><td><p>DIA-NN <a class="footnote-reference brackets" href="#diann" id="id18" role="doc-noteref"><span class="fn-bracket">[</span>26<span class="fn-bracket">]</span></a></p></td>
<td><p>2019 Nov.</p></td>
<td><p>Active</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C &amp; G</p></td>
<td><p>F</p></td>
</tr>
<tr class="row-even"><td><p>DIA tensor <a class="footnote-reference brackets" href="#diat" id="id19" role="doc-noteref"><span class="fn-bracket">[</span>32<span class="fn-bracket">]</span></a></p></td>
<td><p>2020 Oct.</p></td>
<td><p>Not</p></td>
<td><p>I</p></td>
<td><p>/</p></td>
<td><p>C</p></td>
<td><p>F</p></td>
</tr>
<tr class="row-odd"><td><p>MaxDIA <a class="footnote-reference brackets" href="#id37" id="id20" role="doc-noteref"><span class="fn-bracket">[</span>33<span class="fn-bracket">]</span></a></p></td>
<td><p>2021 July</p></td>
<td><p>Active</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C &amp; G</p></td>
<td><p>F</p></td>
</tr>
<tr class="row-even"><td><p>DIAmeter <a class="footnote-reference brackets" href="#id38" id="id21" role="doc-noteref"><span class="fn-bracket">[</span>34<span class="fn-bracket">]</span></a></p></td>
<td><p>2021 July</p></td>
<td><p>Active</p></td>
<td><p>S</p></td>
<td><p>S</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>FIGS <a class="footnote-reference brackets" href="#id39" id="id22" role="doc-noteref"><span class="fn-bracket">[</span>35<span class="fn-bracket">]</span></a></p></td>
<td><p>2021 July</p></td>
<td><p>Not</p></td>
<td><p>L</p></td>
<td><p>S</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>mstc <a class="footnote-reference brackets" href="#id40" id="id23" role="doc-noteref"><span class="fn-bracket">[</span>36<span class="fn-bracket">]</span></a></p></td>
<td><p>2021 July</p></td>
<td><p>Not</p></td>
<td><p>I</p></td>
<td><p>/</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>CsoDIAq <a class="footnote-reference brackets" href="#id41" id="id24" role="doc-noteref"><span class="fn-bracket">[</span>37<span class="fn-bracket">]</span></a></p></td>
<td><p>2021 Sep.</p></td>
<td><p>Active</p></td>
<td><p>L</p></td>
<td><p>S</p></td>
<td><p>C &amp; G</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-even"><td><p>DreamDIA<sup>XMBD</sup> <a class="footnote-reference brackets" href="#id42" id="id25" role="doc-noteref"><span class="fn-bracket">[</span>38<span class="fn-bracket">]</span></a></p></td>
<td><p>2021 Oct.</p></td>
<td><p>Not</p></td>
<td><p>L</p></td>
<td><p>C</p></td>
<td><p>C</p></td>
<td><p>O</p></td>
</tr>
<tr class="row-odd"><td><p>MSFragger-DIA <a class="footnote-reference brackets" href="#msfraggerdia" id="id26" role="doc-noteref"><span class="fn-bracket">[</span>39<span class="fn-bracket">]</span></a></p></td>
<td><p>2020</p></td>
<td><p>Active</p></td>
<td><p>S</p></td>
<td><p>S</p></td>
<td><p>C &amp; G</p></td>
<td><p>F</p></td>
</tr>
</tbody>
</table>
</div>
<div class="admonition note">
<p class="admonition-title">Note</p>
<ul class="simple">
<li><dl class="simple">
<dt><em>Pub date (publication date)</em>: the date the tool was publicly released by publisher</dt><dd><ul>
<li><p><em>Spectronaut</em> is noted by its public release date</p></li>
<li><p><em>PEAKS</em> is noted by the date it supported the analysis of DIA data (PEAKS X in 2018)</p></li>
<li><p><em>MSFragger-DIA</em> is noted by its function-available date</p></li>
</ul>
</dd>
</dl>
</li>
<li><dl class="simple">
<dt><em>Dev stat (development status)</em></dt><dd><ul>
<li><p>Active: has update of any type in last one year (e.g. response of question from authors on GitHub issues, document update, bug fix, and feature update)</p></li>
<li><p>Not: not active</p></li>
<li><p><em>PECAN</em> is re-implemented as Walnut and is being maintained and upgraded</p></li>
</ul>
</dd>
</dl>
</li>
<li><dl class="simple">
<dt><em>Stra. (analysis strategies)</em> (note: some functions for analysis of other data type, like DDA, are not included)</dt><dd><ul>
<li><p>R: spectra reconstruction</p></li>
<li><p>S: sequence-based search</p></li>
<li><p>L: library-based search</p></li>
<li><p>D: de novo sequencing</p></li>
<li><p>I: sequencing-independent</p></li>
</ul>
</dd>
</dl>
</li>
<li><dl class="simple">
<dt><em>x-first (spectrum-/chromatogram-first analysis approaches)</em></dt><dd><ul>
<li><p>S: spectrum-first</p></li>
<li><p>C: chromatogram-first</p></li>
<li><p>N: unknown</p></li>
<li><p>/:</p></li>
<li><p><em>DIA-Umpire</em> and <em>Group-DIA</em> work for two functions: 1. spectra reconstruction, which belongs to neither approach; 2. targeted re-extraction for quantification, which has a chromatogram-first approach</p></li>
<li><p><em>DeepNovo-DIA</em> works for de novo sequencing</p></li>
</ul>
</dd>
</dl>
</li>
<li><dl class="simple">
<dt>(User) interface</dt><dd><ul>
<li><p>C: command line interface (CLI)</p></li>
<li><p>G: graphical user interface (GUI)</p></li>
<li><p>Note: the following software get their GUI support from accompanying tools or platforms – OpenSWATH: TOPPAS, GALAXY; DIA-Umpire: FragPipe; PECAN (Walnut): EncyclopeDIA; MSFragger-DIA: FragPipe</p></li>
</ul>
</dd>
</dl>
</li>
<li><dl class="simple">
<dt><em>Acce. (accessibility)</em></dt><dd><ul>
<li><p>C: commercial</p></li>
<li><p>F: free access</p></li>
<li><p>O: open source (whilst, free access)</p></li>
<li><p>R: need request from authors</p></li>
<li><p>N: not mentioned in the paper</p></li>
</ul>
</dd>
</dl>
</li>
</ul>
</div>
<div class="line-block">
<div class="line"><br /></div>
</div>
</section>
<section id="individuals">
<h2>Individuals<a class="headerlink" href="#individuals" title="Link to this heading">#</a></h2>
<div class="line-block">
<div class="line"><br /></div>
</div>
<section id="ion-accounting">
<h3>Ion Accounting<a class="headerlink" href="#ion-accounting" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="ionaccounting" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id1">1</a><span class="fn-bracket">]</span></span>
<p>Database searching and accounting of multiplexed precursor and product ion spectra from the data independent analysis of simple and complex peptide mixtures. 10.1002/pmic.200800564</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>Ion Accounting</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Guo-Zhong Li et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>Waters Corporation</p></td>
</tr>
<tr class="row-even"><td><p>Publication date</p></td>
<td><p>2009 Mar.</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Spectra reconstruction and seqeunce-based search on extracted precursor-fragment ion pairs</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>spectra reconstruction + seqeunce-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Pre-processing</p></td>
<td><p>Ion detection based on 2D-convolution</p></td>
</tr>
<tr class="row-even"><td><p>Ion grouping criteria</p></td>
<td><p>Apex RT</p></td>
</tr>
<tr class="row-odd"><td><p>Fragment assignment</p></td>
<td><p>One-to-multi (reassigned according to matching quality in sequence-based search step)</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="etiseq">
<h3>ETISEQ<a class="headerlink" href="#etiseq" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id27" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id2">2</a><span class="fn-bracket">]</span></span>
<p>ETISEQ - an algorithm for automated elution time ion sequencing of concurrently fragmented peptides for mass spectrometry-based proteomics. 10.1186/1471-2105-10-244</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>Elution Time Ion Sequencing</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Jason WH Wong et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>Wong Lab</p></td>
</tr>
<tr class="row-even"><td><p>Publication date</p></td>
<td><p>2009 Aug.</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Spectra reconstruction</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>spectra reconstruction</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Pre-processing</p></td>
<td><p>Potential contaminant signal removing</p></td>
</tr>
<tr class="row-even"><td><p>Ion grouping criteria</p></td>
<td><p>PCC and cross-correlation lag</p></td>
</tr>
<tr class="row-odd"><td><p>Fragment assignment</p></td>
<td><ol class="arabic simple">
<li><p>One-to-multi; 2. unmatched fragments are assigned to all precursor</p></li>
</ol>
</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="demux">
<h3>DeMux<a class="headerlink" href="#demux" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id28" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id3">3</a><span class="fn-bracket">]</span></span>
<p>Deconvolution of Mixture Spectra from Ion-Trap Data-Independent-Acquisition Tandem Mass Spectrometry. 10.1021/ac901801b</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DeMux</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Marshall Bern et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>MacCoss Lab</p></td>
</tr>
<tr class="row-even"><td><p>Publication date</p></td>
<td><p>2009 Dec.</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Spectra reconstruction</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>spectra reconstruction</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Pre-processing</p></td>
<td><p>Coarse spectral binning</p></td>
</tr>
<tr class="row-even"><td><p>Ion grouping criteria</p></td>
<td><p>(No precursor requirement) correlation between m/z bins in MS2 spectral map</p></td>
</tr>
<tr class="row-odd"><td><p>Fragment assignment</p></td>
<td><p>One-to-multi, but the cluster center (seed) is dropped in each iteration</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="skyline">
<h3>Skyline<a class="headerlink" href="#skyline" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id29" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id4">4</a><span class="fn-bracket">]</span></span>
<p>Skyline: an open source document editor for creating and analyzing targeted proteomics experiments. 10.1093/bioinformatics/btq054</p>
</aside>
<aside class="footnote brackets" id="skyline2" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>5<span class="fn-bracket">]</span></span>
<p>A framework for installable external tools in Skyline. 10.1093/bioinformatics/btu148</p>
</aside>
<aside class="footnote brackets" id="skyline3" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>6<span class="fn-bracket">]</span></span>
<p>Multiplexed peptide analysis using data-independent acquisition and Skyline. 10.1038/nprot.2015.055</p>
</aside>
<aside class="footnote brackets" id="skyline4" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>7<span class="fn-bracket">]</span></span>
<p>Using Skyline to Analyze Data-Containing Liquid Chromatography, Ion Mobility Spectrometry, and Mass Spectrometry Dimensions. 10.1007/s13361-018-2028-5</p>
</aside>
<aside class="footnote brackets" id="skyline5" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>8<span class="fn-bracket">]</span></span>
<p>The Skyline ecosystem: Informatics for quantitative mass spectrometry proteomics. 10.1002/mas.21540</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>Skyline</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://skyline.ms/project/home/software/Skyline/begin.view">https://skyline.ms/project/home/software/Skyline/begin.view</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/ProteoWizard/pwiz">https://github.com/ProteoWizard/pwiz</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>C#</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Brendan X. MacLean et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>MacCoss Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2010 Apr.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>Peak group</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>(mProphet) linear classifier; (Avant-Garde) combination of sub-scores with fixed parameters</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="ft-arm">
<h3>FT-ARM<a class="headerlink" href="#ft-arm" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="ftarm" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id5">9</a><span class="fn-bracket">]</span></span>
<p>Accurate Peptide Fragment Mass Analysis: Multiplexed Peptide Identification and Quantification. 10.1021/pr2008175</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>FT-ARM</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://brucelab.gs.washington.edu/FT-ARM/index.html">https://brucelab.gs.washington.edu/FT-ARM/index.html</a></p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Chad R. Weisbrod et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Bruce Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2012 Jan.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Sequence-/Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Sequence-/Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>Dot product</p></td>
</tr>
<tr class="row-odd"><td><p>Used prior information</p></td>
<td><p>(optional) fragmentation pattern</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="spectronaut">
<h3>Spectronaut<a class="headerlink" href="#spectronaut" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id30" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id6">10</a><span class="fn-bracket">]</span></span>
<p>Extending the Limits of Quantitative Proteome Profiling with Data-Independent Acquisition and Application to Acetaminophen-Treated Three-Dimensional Liver Microtissues. 10.1074/mcp.M114.044305</p>
</aside>
</aside>
<p>Introduced PTM scoring in version 13:</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="spectronaut2" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>11<span class="fn-bracket">]</span></span>
<p>Rapid and site-specific deep phosphoproteome profiling by data-independent acquisition without the need for spectral libraries. 10.1038/s41467-020-14609-1</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>Spectronaut</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://biognosys.com/software/spectronaut">https://biognosys.com/software/spectronaut</a></p></td>
</tr>
<tr class="row-odd"><td><p>Programming language</p></td>
<td><p>C#</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Roland Bruderer et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>Biognosys AG</p></td>
</tr>
<tr class="row-even"><td><p>Release date</p></td>
<td><p>2013</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Spectra reconstruction and sequence-based search on pseudo spectra; Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="openswath">
<h3>OpenSWATH<a class="headerlink" href="#openswath" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id31" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id7">12</a><span class="fn-bracket">]</span></span>
<p>OpenSWATH enables automated, targeted analysis of data-independent acquisition MS data. 10.1038/nbt.2841</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>OpenSWATH</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="http://openswath.org">http://openswath.org</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/OpenMS/OpenMS">https://github.com/OpenMS/OpenMS</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>C++ &amp; Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Hannes L. Röst et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Aebersold Lab &amp; Röst Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2014 Mar.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>Peak group</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>(PyProphet) XGBoost</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="diana">
<h3>DIANA<a class="headerlink" href="#diana" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id32" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id8">13</a><span class="fn-bracket">]</span></span>
<p>DIANA—algorithmic improvements for analysis of data-independent acquisition MS data. 10.1093/bioinformatics/btu686</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DIANA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://proteomics.immunoprot.lth.se">https://proteomics.immunoprot.lth.se</a></p></td>
</tr>
<tr class="row-odd"><td><p>Programming language</p></td>
<td><p>Scala &amp; Python</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Johan Teleman et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>Levander Lab</p></td>
</tr>
<tr class="row-even"><td><p>Publication date</p></td>
<td><p>2014 Oct.</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>Peak group</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>(PyProphet) XGBoost</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="dia-umpire">
<h3>DIA-Umpire<a class="headerlink" href="#dia-umpire" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diaumpire" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id9">14</a><span class="fn-bracket">]</span></span>
<p>DIA-Umpire: comprehensive computational framework for data-independent acquisition proteomics. 10.1038/nmeth.3255</p>
</aside>
<aside class="footnote brackets" id="diaumpire2" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>15<span class="fn-bracket">]</span></span>
<p>Untargeted, spectral library-free analysis of data-independent acquisition proteomics data generated using Orbitrap mass spectrometers. 10.1002/pmic.201500526</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DIA-Umpire</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://diaumpire.nesvilab.org">https://diaumpire.nesvilab.org</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/Nesvilab/DIA-Umpire">https://github.com/Nesvilab/DIA-Umpire</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Java</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Chih-Chiang Tsou et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Nesvizhskii Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2015 Jan.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Spectra reconstruction; library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>spectra reconstruction</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Pre-processing</p></td>
<td><p>Interpolation and unimodal peak splitting</p></td>
</tr>
<tr class="row-even"><td><p>Ion grouping criteria</p></td>
<td><ol class="arabic simple">
<li><p>summed mass of two fragments equals to precursor mass; 2. precursor-fragment PCC rank and apex RT</p></li>
</ol>
</td>
</tr>
<tr class="row-odd"><td><p>Fragment assignment</p></td>
<td><p>One-to-multi</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="swathprophet">
<h3>SWATHProphet<a class="headerlink" href="#swathprophet" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id33" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id10">16</a><span class="fn-bracket">]</span></span>
<p>Automated Validation of Results and Removal of Fragment Ion Interferences in Targeted Analysis of Data-independent Acquisition Mass Spectrometry (MS) using SWATHProphet. 10.1074/mcp.O114.044917</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>SWATHProphet</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="http://tools.proteomecenter.org/wiki/index.php?title=Software:SWATHProphet">http://tools.proteomecenter.org/wiki/index.php?title=Software:SWATHProphet</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://sourceforge.net/projects/sashimi/files/">https://sourceforge.net/projects/sashimi/files/</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Perl</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Andrew Keller et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Moritz Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2015 Feb.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>Peak group</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>mProphet-like linear classifier</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="group-dia">
<h3>Group-DIA<a class="headerlink" href="#group-dia" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="groupdia" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id11">17</a><span class="fn-bracket">]</span></span>
<p>Group-DIA: analyzing multiple data-independent acquisition mass spectrometry data files. 10.1038/nmeth.3593</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>Group-DIA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="http://yuanyueli.github.io/group-dia/">http://yuanyueli.github.io/group-dia/</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/YuanyueLi/group-dia">https://github.com/YuanyueLi/group-dia</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>C++</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Yuanyue Li et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Han Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2015 Oct.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Spectra reconstruction; library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>spectra reconstruction</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Pre-processing</p></td>
<td><p>Cross-run spectra alignment</p></td>
</tr>
<tr class="row-even"><td><p>Ion grouping criteria</p></td>
<td><p>k-means-based precursor-fragment clustering</p></td>
</tr>
<tr class="row-odd"><td><p>Fragment assignment</p></td>
<td><p>One-to-one and high precursor-fragment correlation-first</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="msplit-dia">
<h3>MSPLIT-DIA<a class="headerlink" href="#msplit-dia" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="msplitdia" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id12">18</a><span class="fn-bracket">]</span></span>
<p>MSPLIT-DIA: sensitive peptide identification for data-independent acquisition. 10.1038/nmeth.3655</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>MSPLIT-DIA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="http://proteomics.ucsd.edu/software-tools/msplit-dia">http://proteomics.ucsd.edu/software-tools/msplit-dia</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p>/</p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Java</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Jian Wang et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Bandeira Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2015 Nov.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>Cosine similarity</p></td>
</tr>
<tr class="row-odd"><td><p>Used prior information</p></td>
<td><p>Fragmentation pattern; (optional) RT</p></td>
</tr>
<tr class="row-even"><td><p>Peak-picking of shared m/z</p></td>
<td><p>High quality-first by SSM similarity</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="pecan-walnut">
<h3>PECAN (Walnut)<a class="headerlink" href="#pecan-walnut" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="pecan" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id13">19</a><span class="fn-bracket">]</span></span>
<p>PECAN: library-free peptide detection for data-independent acquisition tandem mass spectrometry data. 10.1038/nmeth.4390</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>PECAN (Walnut in Encyclopedia platform)</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p>Walnut: <a class="reference external" href="https://bitbucket.org/searleb/encyclopedia/wiki/Home">https://bitbucket.org/searleb/encyclopedia/wiki/Home</a>; PECAN: <a class="reference external" href="https://bitbucket.org/maccosslab/pecan/wiki/Home">https://bitbucket.org/maccosslab/pecan/wiki/Home</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p>Walnut: <a class="reference external" href="https://bitbucket.org/searleb/encyclopedia/src">https://bitbucket.org/searleb/encyclopedia/src</a>; PECAN: <a class="reference external" href="https://bitbucket.org/maccosslab/pecan/src">https://bitbucket.org/maccosslab/pecan/src</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Walnut: Java; PECAN: Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Ying S. Ting et al. (re-implemented by Brian C. Searle in Encyclopedia platform)</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>MacCoss Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2017 Aug.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Sequence-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Sequence-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>Background score-subtracted dot product</p></td>
</tr>
<tr class="row-odd"><td><p>Matching restriction</p></td>
<td><p>Reduce fragment matching importance based on frequency</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="specter">
<h3>Specter<a class="headerlink" href="#specter" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id34" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id14">20</a><span class="fn-bracket">]</span></span>
<p>Specter: linear deconvolution for targeted analysis of data-independent acquisition mass spectrometry proteomics. 10.1038/nmeth.4643</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>Specter</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/rpeckner-broad/Specter">https://github.com/rpeckner-broad/Specter</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/rpeckner-broad/Specter">https://github.com/rpeckner-broad/Specter</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Ryan Peckner et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Jaffe Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2018 Apr.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>Coefficients of linear combination of library spectra</p></td>
</tr>
<tr class="row-odd"><td><p>Used prior information</p></td>
<td><p>Fragmentation pattern; RT</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="encyclopedia">
<h3>EncyclopeDIA<a class="headerlink" href="#encyclopedia" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id35" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id15">21</a><span class="fn-bracket">]</span></span>
<p>Chromatogram libraries improve peptide detection and quantification by data independent acquisition mass spectrometry. 10.1038/s41467-018-07454-w</p>
</aside>
<aside class="footnote brackets" id="encyclopedia2" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>22<span class="fn-bracket">]</span></span>
<p>Generating high quality libraries for DIA MS with empirically corrected peptide predictions. 10.1038/s41467-020-15346-1</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>EncyclopeDIA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://bitbucket.org/searleb/encyclopedia/wiki/Home">https://bitbucket.org/searleb/encyclopedia/wiki/Home</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://bitbucket.org/searleb/encyclopedia/src">https://bitbucket.org/searleb/encyclopedia/src</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Java</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Brian C. Searle et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>MacCoss Lab &amp; Searle Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2018 Dec.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>product of summed correlation-weighted dot products and the factorial of the number of matched ions</p></td>
</tr>
<tr class="row-odd"><td><p>Used prior information</p></td>
<td><p>Fragmentation pattern; RT</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="deepnovo-dia">
<h3>DeepNovo-DIA<a class="headerlink" href="#deepnovo-dia" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="deepnovodia" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id16">23</a><span class="fn-bracket">]</span></span>
<p>Deep learning enables de novo peptide sequencing from data-independent-acquisition mass spectrometry. 10.1038/s41592-018-0260-3</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DeepNovo-DIA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/nh2tran/DeepNovo-DIA">https://github.com/nh2tran/DeepNovo-DIA</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/nh2tran/DeepNovo-DIA">https://github.com/nh2tran/DeepNovo-DIA</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Ngoc Hieu Tran et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Li Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2018 Dec.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>De novo sequencing</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="peaks">
<h3>PEAKS<a class="headerlink" href="#peaks" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id36" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id17">24</a><span class="fn-bracket">]</span></span>
<p>PEAKS: powerful software for peptide de novo sequencing by tandem mass spectrometry. 10.1002/rcm.1196</p>
</aside>
<aside class="footnote brackets" id="peaks2" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>25<span class="fn-bracket">]</span></span>
<p>A streamlined platform for analyzing tera-scale DDA and DIA mass spectrometry data enables highly sensitive immunopeptidomics. 10.1038/s41467-022-30867-7</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>PEAKS</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://www.bioinfor.com/peaks-software">https://www.bioinfor.com/peaks-software</a></p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Bin Ma et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Bioinformatics Solutions Inc</p></td>
</tr>
<tr class="row-odd"><td><p>Support date</p></td>
<td><p>2018</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Sequence-based search; library-based search; de novo sequencing</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="dia-nn">
<h3>DIA-NN<a class="headerlink" href="#dia-nn" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<p>Original:</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diann" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id18">26</a><span class="fn-bracket">]</span></span>
<p>DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. 2020. 10.1038/s41592-019-0638-x</p>
</aside>
</aside>
<p>Introduced new analysis strategies for identification, scoring, and quantification of Scanning SWATH data:</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diann2" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>27<span class="fn-bracket">]</span></span>
<p>Ultra-fast proteomics with Scanning SWATH. 2021. 10.1038/s41587-021-00860-4</p>
</aside>
</aside>
<p>Introduced the PTM scoring function with ubiquitinome DIA data as examples:</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diann3" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>28<span class="fn-bracket">]</span></span>
<p>Time-resolved in vivo ubiquitinome profiling by DIA-MS reveals USP7 targets on a proteome-wide scale. 2021. 10.1038/s41467-021-25454-1</p>
</aside>
</aside>
<p>Introduced the integration of DIA-NN with FragPipe platform, and the analysis details for diaPASEF data:</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diann4" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>29<span class="fn-bracket">]</span></span>
<p>dia-PASEF data analysis using FragPipe and DIA-NN for deep proteomics of low sample amounts. 2022. 10.1038/s41467-022-31492-0</p>
</aside>
</aside>
<p>Introduced new analysis strategies for identification, scoring, and quantification of plexDIA data (non-isobaric label):</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diann5" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>30<span class="fn-bracket">]</span></span>
<p>Increasing the throughput of sensitive proteomics by plexDIA. 2023. 10.1038/s41587-022-01389-w</p>
</aside>
</aside>
<p>Introduced QuantUMS for improved quantification:</p>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diann6" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span>31<span class="fn-bracket">]</span></span>
<p>QuantUMS: uncertainty minimisation enables confident quantification in proteomics. 2023. 10.1101/2023.06.20.545604</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DIA-NN</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/vdemichev/diann">https://github.com/vdemichev/diann</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/vdemichev/diann">https://github.com/vdemichev/diann</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>C++</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Vadim Demichev et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Ralser Lab &amp; Demichev Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2019 Nov.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>One best peak</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>linear classifier and fully-connected neural network</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="dia-tensor">
<h3>DIA tensor<a class="headerlink" href="#dia-tensor" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="diat" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id19">32</a><span class="fn-bracket">]</span></span>
<p>Phenotype Classification using Proteome Data in a Data-Independent Acquisition Tensor Format. 2020. 10.1021/jasms.0c00254</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DIA tensor</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/guomics-lab/DIAtensor">https://github.com/guomics-lab/DIAtensor</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/guomics-lab/DIAtensor">https://github.com/guomics-lab/DIAtensor</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Fangfei Zhang et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Guo Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2020 Oct.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Sequencing-independent</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="maxdia">
<h3>MaxDIA<a class="headerlink" href="#maxdia" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id37" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id20">33</a><span class="fn-bracket">]</span></span>
<p>MaxDIA enables library-based and library-free data-independent acquisition proteomics. 2021. 10.1038/s41587-021-00968-7</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>MaxDIA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://maxquant.org/">https://maxquant.org/</a></p></td>
</tr>
<tr class="row-odd"><td><p>Programming language</p></td>
<td><p>C#</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Pavel Sinitcyn et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>Cox Lab</p></td>
</tr>
<tr class="row-even"><td><p>Publication date</p></td>
<td><p>2021 July</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>Peak group</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>XGBoost</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="diameter">
<h3>DIAmeter<a class="headerlink" href="#diameter" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id38" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id21">34</a><span class="fn-bracket">]</span></span>
<p>DIAmeter: matching peptides to data-independent acquisition mass spectrometry data. 2021. 10.1093/bioinformatics/btab284</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DIAmeter</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="http://crux.ms/">http://crux.ms/</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/crux-toolkit/crux-toolkit">https://github.com/crux-toolkit/crux-toolkit</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>C++</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Yang Young Lu et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Noble Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2021 July</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Sequence-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Sequence-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>XCorr</p></td>
</tr>
<tr class="row-odd"><td><p>Matching restriction</p></td>
<td><ol class="arabic simple">
<li><p>Maximum PSMs per charge state per spectrum; 2. PSM reduction based on combined sub-scores compared to PSM with highest primary score</p></li>
</ol>
</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="figs">
<h3>FIGS<a class="headerlink" href="#figs" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id39" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id22">35</a><span class="fn-bracket">]</span></span>
<p>FIGS: Featured Ion-Guided Stoichiometry for Data-Independent Proteomics through Dynamic Deconvolution. 10.1021/acs.jproteome.1c00438</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>FIGS</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/FangYan-USTC/FIGS">https://github.com/FangYan-USTC/FIGS</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/FangYan-USTC/FIGS">https://github.com/FangYan-USTC/FIGS</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Yan Fang et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Zeng Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2021 July</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>Coefficients of linear combination of library spectra</p></td>
</tr>
<tr class="row-odd"><td><p>Used prior information</p></td>
<td><p>Fragmentation pattern; RT</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="mstc">
<h3>mstc<a class="headerlink" href="#mstc" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id40" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id23">36</a><span class="fn-bracket">]</span></span>
<p>On the feasibility of deep learning applications using raw mass spectrometry data. 2021. 10.1093/bioinformatics/btab311</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>mass_spec_trans_coding</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/PhosphorylatedRabbits/mass_spec_trans_coding">https://github.com/PhosphorylatedRabbits/mass_spec_trans_coding</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/PhosphorylatedRabbits/mass_spec_trans_coding">https://github.com/PhosphorylatedRabbits/mass_spec_trans_coding</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Joris Cadow et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Martínez Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2021 July</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Sequencing-independent</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="csodiaq">
<h3>CsoDIAq<a class="headerlink" href="#csodiaq" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id41" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id24">37</a><span class="fn-bracket">]</span></span>
<p>CsoDIAq Software for Direct Infusion Shotgun Proteome Analysis. 10.1038/nmeth.3655</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>CsoDIAq</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/CCranney/CsoDIAq">https://github.com/CCranney/CsoDIAq</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/CCranney/CsoDIAq">https://github.com/CCranney/CsoDIAq</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Caleb W. Cranney and Jesse G. Meyer</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Meyer Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2021 Sep.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>Cosine similarity</p></td>
</tr>
<tr class="row-odd"><td><p>Used prior information</p></td>
<td><p>Fragmentation pattern</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="dreamdiaxmbd">
<h3>DreamDIA<sup>XMBD</sup><a class="headerlink" href="#dreamdiaxmbd" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="id42" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id25">38</a><span class="fn-bracket">]</span></span>
<p>Deep representation features from DreamDIA<sup>XMBD</sup> improve the analysis of data-independent acquisition proteomics. 2021. 10.1038/s42003-021-02726-6</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>DreamDIA<sup>XMBD</sup></p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://github.com/xmuyulab/DreamDIA-XMBD">https://github.com/xmuyulab/DreamDIA-XMBD</a></p></td>
</tr>
<tr class="row-odd"><td><p>Source code</p></td>
<td><p><a class="reference external" href="https://github.com/xmuyulab/DreamDIA-XMBD">https://github.com/xmuyulab/DreamDIA-XMBD</a></p></td>
</tr>
<tr class="row-even"><td><p>Programming language</p></td>
<td><p>Python</p></td>
</tr>
<tr class="row-odd"><td><p>Authors</p></td>
<td><p>Mingxuan Gao et al.</p></td>
</tr>
<tr class="row-even"><td><p>Lab &amp; Company</p></td>
<td><p>Han Lab &amp; Yu Lab</p></td>
</tr>
<tr class="row-odd"><td><p>Publication date</p></td>
<td><p>2021 Oct.</p></td>
</tr>
<tr class="row-even"><td><p>Analysis strategies</p></td>
<td><p>Library-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Library-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Chromatogram-first</p></td>
</tr>
<tr class="row-even"><td><p>Minimum scoring unit</p></td>
<td><p>Raw XICs</p></td>
</tr>
<tr class="row-odd"><td><p>Scoring model</p></td>
<td><p>LSTM neural network and XGBoost</p></td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="msfragger-dia">
<h3>MSFragger-DIA<a class="headerlink" href="#msfragger-dia" title="Link to this heading">#</a></h3>
<p>back to <a class="reference internal" href="#table">table</a></p>
<dl class="field-list simple">
<dt class="field-odd">Reference<span class="colon">:</span></dt>
<dd class="field-odd"><p></p></dd>
</dl>
<aside class="footnote-list brackets">
<aside class="footnote brackets" id="msfraggerdia" role="doc-footnote">
<span class="label"><span class="fn-bracket">[</span><a role="doc-backlink" href="#id26">39</a><span class="fn-bracket">]</span></span>
<p>Analysis of DIA proteomics data using MSFragger-DIA and FragPipe computational platform. 2023. 10.1038/s41467-023-39869-5</p>
</aside>
</aside>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Full name</p></td>
<td><p>MSFragger-DIA</p></td>
</tr>
<tr class="row-even"><td><p>Home page</p></td>
<td><p><a class="reference external" href="https://msfragger.nesvilab.org">https://msfragger.nesvilab.org</a> &amp; <a class="reference external" href="https://github.com/Nesvilab/MSFragger">https://github.com/Nesvilab/MSFragger</a></p></td>
</tr>
<tr class="row-odd"><td><p>Programming language</p></td>
<td><p>Java</p></td>
</tr>
<tr class="row-even"><td><p>Authors</p></td>
<td><p>Fengchao Yu et al.</p></td>
</tr>
<tr class="row-odd"><td><p>Lab &amp; Company</p></td>
<td><p>Nesvizhskii Lab</p></td>
</tr>
<tr class="row-even"><td><p>Support date</p></td>
<td><p>2020</p></td>
</tr>
<tr class="row-odd"><td><p>Analysis strategies</p></td>
<td><p>Sequence-based search</p></td>
</tr>
</tbody>
</table>
</div>
<dl class="field-list simple">
<dt class="field-odd">Function<span class="colon">:</span></dt>
<dd class="field-odd"><p>Sequence-based search</p>
</dd>
</dl>
<div class="table-wrapper docutils container">
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Analysis approach</p></td>
<td><p>Spectrum-first</p></td>
</tr>
<tr class="row-even"><td><p>SSM score</p></td>
<td><p>MSFragger hyperscore</p></td>
</tr>
<tr class="row-odd"><td><p>Matching restriction</p></td>
<td><ol class="arabic simple">
<li><p>Maximum PSMs per spectrum; 2. High PSM score-first signal picking</p></li>
</ol>
</td>
</tr>
</tbody>
</table>
</div>
</section>
</section>
</section>

        </article>
      </div>
      <footer>
        
        <div class="related-pages">
          <a class="next-page" href="Datasets_Benchmark.html">
              <div class="page-info">
                <div class="context">
                  <span>Next</span>
                </div>
                <div class="title">Benchmark data sets</div>
              </div>
              <svg class="furo-related-icon"><use href="#svg-arrow-right"></use></svg>
            </a>
          <a class="prev-page" href="Acquisition_Unconventional.html">
              <svg class="furo-related-icon"><use href="#svg-arrow-right"></use></svg>
              <div class="page-info">
                <div class="context">
                  <span>Previous</span>
                </div>
                
                <div class="title">Unconventional methods</div>
                
              </div>
            </a>
        </div>
        <div class="bottom-of-page">
          <div class="left-details">
            <div class="copyright">
                Copyright &#169; 2023, Shui Lab
            </div>
            Made with <a href="https://www.sphinx-doc.org/">Sphinx</a> and <a class="muted-link" href="https://pradyunsg.me">@pradyunsg</a>'s
            
            <a href="https://github.com/pradyunsg/furo">Furo</a>
            
          </div>
          <div class="right-details">
            <div class="icons">
              <a class="muted-link " href="https://github.com/Shui-Group/DIAReviewAppendix" aria-label="GitHub">
                <svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 16 16">
                    <path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path>
                </svg>
            </a>
              
            </div>
          </div>
        </div>
        
      </footer>
    </div>
    <aside class="toc-drawer">
      
      
      <div class="toc-sticky toc-scroll">
        <div class="toc-title-container">
          <span class="toc-title">
            On this page
          </span>
        </div>
        <div class="toc-tree-container">
          <div class="toc-tree">
            <ul>
<li><a class="reference internal" href="#">DIA data analysis</a><ul>
<li><a class="reference internal" href="#overall-summary">Overall summary</a></li>
<li><a class="reference internal" href="#individuals">Individuals</a><ul>
<li><a class="reference internal" href="#ion-accounting">Ion Accounting</a></li>
<li><a class="reference internal" href="#etiseq">ETISEQ</a></li>
<li><a class="reference internal" href="#demux">DeMux</a></li>
<li><a class="reference internal" href="#skyline">Skyline</a></li>
<li><a class="reference internal" href="#ft-arm">FT-ARM</a></li>
<li><a class="reference internal" href="#spectronaut">Spectronaut</a></li>
<li><a class="reference internal" href="#openswath">OpenSWATH</a></li>
<li><a class="reference internal" href="#diana">DIANA</a></li>
<li><a class="reference internal" href="#dia-umpire">DIA-Umpire</a></li>
<li><a class="reference internal" href="#swathprophet">SWATHProphet</a></li>
<li><a class="reference internal" href="#group-dia">Group-DIA</a></li>
<li><a class="reference internal" href="#msplit-dia">MSPLIT-DIA</a></li>
<li><a class="reference internal" href="#pecan-walnut">PECAN (Walnut)</a></li>
<li><a class="reference internal" href="#specter">Specter</a></li>
<li><a class="reference internal" href="#encyclopedia">EncyclopeDIA</a></li>
<li><a class="reference internal" href="#deepnovo-dia">DeepNovo-DIA</a></li>
<li><a class="reference internal" href="#peaks">PEAKS</a></li>
<li><a class="reference internal" href="#dia-nn">DIA-NN</a></li>
<li><a class="reference internal" href="#dia-tensor">DIA tensor</a></li>
<li><a class="reference internal" href="#maxdia">MaxDIA</a></li>
<li><a class="reference internal" href="#diameter">DIAmeter</a></li>
<li><a class="reference internal" href="#figs">FIGS</a></li>
<li><a class="reference internal" href="#mstc">mstc</a></li>
<li><a class="reference internal" href="#csodiaq">CsoDIAq</a></li>
<li><a class="reference internal" href="#dreamdiaxmbd">DreamDIA<sup>XMBD</sup></a></li>
<li><a class="reference internal" href="#msfragger-dia">MSFragger-DIA</a></li>
</ul>
</li>
</ul>
</li>
</ul>

          </div>
        </div>
      </div>
      
      
    </aside>
  </div>
</div><script src="_static/documentation_options.js?v=837179f8"></script>
    <script src="_static/doctools.js?v=888ff710"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/scripts/furo.js?v=32e29ea5"></script>
    <script src="_static/tabs.js?v=3ee01567"></script>
    </body>
</html>